Source/Other/Extensions/PopupButtonExtensions.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PopupButtonCellExtension.swift
+//  Sequel Ace
+//
+//  Created by James on 25/1/2021.
+//  Copyright © 2021 Sequel-Ace. All rights reserved.
+//
+
+import AppKit
```

Comment:
I'm still not sure if it's import AppKit or imporr Cocoa 😄

Reply:
`import Cocoa` imports:

```swift
import AppKit
import CoreData
import Foundation
```